src/content/blog/kai-deep-dive-2024/index.mdx

```diff
@@ -0,0 +1,170 @@
+---
+author: jwmatthews
+date: 2024-05-07T07:00:00Z
+description: >-
+  Konveyor AI (Kai) Introduction
+featured: true
+image: header.png
+tags:
+  - Konveyor
+  - Kai
+  - GenAI
+  - AppModernization
+title: Kai - Generative AI Applied to Application Modernization
+---
+
+**Author**: [John Matthews](https://github.com/jwmatthews)
+
+### Overview
+
+Konveyor AI or "Kai", is an early effort of Generative AI applied to Application Modernization being explored under the Konveyor Ecosystem at [konveyor-ecosystem/kai](https://github.com/konveyor-ecosystem/kai).
+
+Kai implements a Retrieval Augmented Generation (RAG) approach that leverages data from Konveyor to help generate code suggestions to aid migrating legacy code bases to a different technology. The intent of this RAG approach is to shape the code suggestions to be similar to how an organization has solved problems in the past, without additional fine-tuning of the model.
```

Comment:
suggestion: Add a link to RAG - https://www.promptingguide.ai/techniques/rag 
(feel free to ignore if you think its self-explanatory)

Reply:
Updated